test/integration/golden_response/search_statvar/female.json

```diff
@@ -4003,4003 +4003,4003 @@
     },
     {
-      "name": "Person by Age = 18 - 24 Years, College or Graduate School Enrollment = Enrolled in Private College or Graduate School, Gender = Female",
+      "name": "Person With Age = 18 - 24 Years, College or Graduate School Enrollment = Enrolled in Private College or Graduate School, Gender = Female",
       "dcid": "dc/g/Person_Age-Years18To24_CollegeOrGraduateSchoolEnrollment-EnrolledInPrivateCollegeOrGraduateSchool_Gender-Female"
     },
     {
-      "name": "Person by Age = 18 - 24 Years, College or Graduate School Enrollment = Enrolled in Public College or Graduate School, Gender = Female",
+      "name": "Person With Age = 18 - 24 Years, College or Graduate School Enrollment = Enrolled in Public College or Graduate School, Gender = Female",
       "dcid": "dc/g/Person_Age-Years18To24_CollegeOrGraduateSchoolEnrollment-EnrolledInPublicCollegeOrGraduateSchool_Gender-Female"
     },
     {
-      "name": "Person by Age = 18 - 24 Years, College or Graduate School Enrollment = Not Enrolled in College or Graduate School, Gender = Female",
+      "name": "Person With Age = 18 - 24 Years, College or Graduate School Enrollment = Not Enrolled in College or Graduate School, Gender = Female",
       "dcid": "dc/g/Person_Age-Years18To24_CollegeOrGraduateSchoolEnrollment-NotEnrolledInCollegeOrGraduateSchool_Gender-Female"
     },
     {
-      "name": "Person by Age = 18 - 24 Years, College or Graduate School Enrollment, Gender = Female",
+      "name": "Person With Age = 18 - 24 Years, College or Graduate School Enrollment, Gender = Female",
       "dcid": "dc/g/Person_Age-Years18To24_CollegeOrGraduateSchoolEnrollment_Gender-Female"
     },
     {
-      "name": "Person by Age = 18 - 24 Years, Educational Attainment = 9 Th To 12 Th Grade No Diploma, Gender = Female",
+      "name": "Person With Age = 18 - 24 Years, Educational Attainment = 9 Th To 12 Th Grade No Diploma, Gender = Female",
       "dcid": "dc/g/Person_Age-Years18To24_EducationalAttainment-9ThTo12ThGradeNoDiploma_Gender-Female"
     },
     {
-      "name": "Person by Age = 18 - 24 Years, Educational Attainment = Associates Degree, Gender = Female",
+      "name": "Person With Age = 18 - 24 Years, Educational Attainment = Associates Degree, Gender = Female",
       "dcid": "dc/g/Person_Age-Years18To24_EducationalAttainment-AssociatesDegree_Gender-Female"
     },
     {
-      "name": "Person by Age = 18 - 24 Years, Educational Attainment = Bachelors Degree or Higher, Gender = Female",
+      "name": "Person With Age = 18 - 24 Years, Educational Attainment = Bachelors Degree or Higher, Gender = Female",
       "dcid": "dc/g/Person_Age-Years18To24_EducationalAttainment-BachelorsDegreeOrHigher_Gender-Female"
     },
     {
-      "name": "Person by Age = 18 - 24 Years, Educational Attainment = Bachelors Degree, Gender = Female",
+      "name": "Person With Age = 18 - 24 Years, Educational Attainment = Bachelors Degree, 
```

Comment:
should this be lower case?

Reply:
With seems capitalized - https://screenshot.googleplex.com/8tzhjgbn7M7ELgA

Reply:
I see, anything longer than 3 letters are Cap